clang/lib/Headers/emmintrin.h

```diff
@@ -513,7 +526,7 @@ static __inline__ __m128d __DEFAULT_FN_ATTRS _mm_cmpge_pd(__m128d __a,
 ///    operand are ordered with respect to those in the second operand.
 ///
 ///    A pair of double-precision values are "ordered" with respect to each
```

Comment:
Line 528: Remove the quotation marks around "ordered"? What do they add? Note that there are other instances of this (as well as "unordered" where the quotation marks are inserted) that I have not commented on.

Reply:
Done